metals/src/main/scala/scala/meta/internal/metals/Embedded.scala

```diff
@@ -76,6 +79,30 @@ final class Embedded(
     Files.copy(embeddedBloopClient, out)
     AbsolutePath(out)
   }
+
+  val isScalafmtJars = new AtomicBoolean(false)
+  lazy val scalafmtJars: Option[URLClassLoader] = {
+    isScalafmtJars.set(true)
+    val promise = Promise[Unit]()
+    statusBar.trackFuture(s"${icons.sync}Downloading Scalafmt", promise.future)
+    try {
+      Some(
+        Embedded.newScalafmtClassloader(
+          userConfig().scalafmtVersion.getOrElse("1.5.1")
```

Comment:
The default could possibly be moved somewhere else, but I didn't know where.
Maybe in `BuildInfo`? /cc @olafurpg